surveyor.py

```diff
@@ -18,6 +19,11 @@
 # list of all the different products we support
 @click.option("--threathunter", 'product', help="Use this to use Cb ThreatHunter.", flag_value="cbth", default=False)
 @click.option("--response", 'product', help="Use this to use Cb Response.", flag_value="cbr", default=True)
+@click.option("--defender", 'product', help="Use this to query Microsoft Defender for Endpoints", flag_value="defender", default=False)
```

Comment:
Yes this was intentional, due to Defenders renaming from ATP. If people are still calling it by the previous name of ATP  it would still run (also less letters and option to have a typo).